reframe/utility/__init__.py

```diff
@@ -69,7 +74,7 @@ def _do_import_module_from_file(filename, module_name=None):
     return module
 
 
-def import_module_from_file(filename, force=False):
+def import_module_from_file(filename, force=False, unique_name=False):
```

Comment:
I am giving second thought as of whether we need an option here or use the proposed behaviour always. This function allows you to load a module in a non-standard way, i.e., outside a package, and it tries to create proper integration with `sys.modules` and do proper caching. So I believe the option is not needed and we always do the hashing.